### ISSUE TITLE
pytorch to caffe, how to add a linear layer on slim.py

### ISSUE THREAD

**Issue by youngliuxx** [miaow1988/ShuffleNet_V2_pytorch_caffe#20]
I use your code train a test model, just change the convolution layer to a linear layer, and a linear layer to the slim.py for conversion. When trying to convert the torch model to the caffe model, the mean diff is large.

I want to convert pytorch to caffe and support the linear layer. What other work needs to be done?

my codes:

```Python
# slim.py
if isinstance(m, nn.Linear):
    if m.bias is None:
        param = [dict(lr_mult=1, decay_mult=1)]
    else:
        param = [dict(lr_mult=1, decay_mult=1), dict(lr_mult=1, decay_mult=0)]
    inner_product_param = dict(
        num_output = m.out_features,
        bias_term = (m.bias is not None),
        weight_filler = dict(type='msra'),
    )
    layer = L.InnerProduct(
        layer,
        param=param,
        inner_product_param = inner_product_param,
    )
    caffe_net.tops[m.g_name] = layer
    return layer
```

**Comment by liwei46** [comment]
Hello,

You also need to add the following code:

        if isinstance(m, nn.Linear):
            print('convert FC:', name, m.g_name, m)
            caffe_net.params[m.g_name][0].data[...] = m.weight.data.cpu().numpy()
            if m.bias is not None:
                caffe_net.params[m.g_name][1].data[...] = m.bias.data.cpu().numpy()

in function convert_pytorch_to_caffe in slim.py

hope it helps. 
Regards,
